.github/workflows/test.yml

```diff
@@ -46,10 +46,18 @@ jobs:
           POSTGRES_DB: odoo
         ports:
           - 5432:5432
+    env:
+      ADDONS_PATH: "/opt/odoo/addons,pci-private"
```

Comment:
Actually, it is wrong to have a dependency on a private module.  We should move such modules to pci-private, instead of making changes here.